### PR TITLE
Post ServicePaths while Registering to SC

### PR DESCRIPTION
**Pull request by thandayuthapani**
Post ServicePaths while Registering to SC ( https://github.com/go-chassis/go-chassis/issues/432 )

### CODE REVIEW

**Comment by tianxiaoliang** [comment]
https://go-chassis.readthedocs.io/en/latest/user-guides/microservice.html

File: core/registry/struct.go [hunk 1]
```diff
@@ -16,6 +17,12 @@ type MicroService struct {
 	Alias       string
 }
 
+// ServicePathStruct has info about service path and property
+type ServicePathStruct struct {
```
Comment (tianxiaoliang) [inline core/registry/struct.go]: change to ServicePath

File: docs/user-guides/microservice.md [hunk 2]
```diff
@@ -40,4 +43,8 @@ service_description:
     project: X1
   instance_properties:
     nodeIP: 192.168.0.111
+  paths:
+  - path: /rest/demoservice
+    property:
+      checksession: true
```
Comment (tianxiaoliang) [inline docs/user-guides/microservice.md]: what that checksession means

Comment (daqingshu) [reply docs/user-guides/microservice.md]: checksession just a demo  property, SC will store it

**Pull request merged**